tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py38,py39,py310
+
+[testenv]
+# install pytest in the virtualenv where commands will be executed
+deps = pytest
+commands =
+    # NOTE: you can run any command line tool here - not just tests
+    python parentheses_checker/__init__.py
+    pytest tests/test.py
```

Comment:
I recommend always adding a new-line at the end of all files.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

You can configure most editors/IDEs to do that for you when saving the file.